src/pom.xml

```diff
@@ -55,9 +55,8 @@
         <module>repairnator-pipeline</module>
         <module>repairnator-core</module>
         <module>repairnator-realtime</module>
-				<module>repairnator-jenkins-plugin</module>
```

Comment:
why do you remove the jenkins plugin?

Reply:
I got warning about the pom being malformed due to the Jenkins plugin pom has another parent pom than repairnator-core like other projects. I suspect it also caused a bug discovered when I tried to update Sorald some time ago.

Reply:
The problem is that if we remove this, CI does not compile and test the jenkins plugin anymore.

The warning is not important, we can put this line back.

Reply:
Then I'll put it back for now. If there are any issue, we can just have a travis build step for the Jenkins with custom script different from the other build step. :)